endpoints/_endpointscfg_setup.py

```diff
@@ -81,18 +81,18 @@ def _SetupPaths():
   if sdk_path:
     sys.path.append(sdk_path)
     try:
-      import wrapper_util  # pylint: disable=g-import-not-at-top
-      paths = wrapper_util.Paths(sdk_path)
-      additional_paths = paths.v1_extra_paths + paths.endpointscfg_extra_paths
-      sys.path = additional_paths + sys.path
+      import dev_appserver  # pylint: disable=g-import-not-at-top
+      dev_appserver.fix_sys_path()
     except ImportError:
       logging.warning(_IMPORT_ERROR_WARNING)
   else:
     logging.warning(_NOT_FOUND_WARNING)
 
   # Add the path above this directory, so we can import the endpoints package
   # from the user's app code (rather than from another, possibly outdated SDK).
-  sys.path = [os.path.dirname(os.path.dirname(__file__))] + sys.path
+  # pylint: disable=g-import-not-at-top
```

Comment:
Can't. It's 81 characters following style guide, which is two spaces before an inline comment.